### PR TITLE
File added: online_book_store_documentation.pdf

### PR DESCRIPTION
**Pull request by vijaya1811**
done changes and pushed modified file

### CODE REVIEW

**Review by shubhamdalal49 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

File: vijaya/html/assignment6/assign6que2.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                <legend>Academic Details:</legend>
+                <b>Highest Qualification:</b>
+                <select>
+                    <option value="diploma">Diploma In Competer Science</option>
+                    <option value="graduate">Science Graduate</option>
+                    <option value="bacheler">Bacheler In Enggineering</option>
+                    <option value="mcs">Master In Computer Science</option>
+                    <option value="master">Master In Enggineering</option>
+                    <option value="other">Other</option></select
+                ><br />
```
Comment (shubhamdalal49) [inline vijaya/html/assignment6/assign6que2.html]: check indentation in select tag

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

File: vijaya/html/assignment5/question1.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  h2{
+   color:white;
+    text-align:right;
+  }
+  a:link {
+text-decoration:blueviolet;
+}
+ul{
+ list-style-type: square;
+}
```
Comment (Pawan1005) [inline vijaya/html/assignment5/question1.css:35]: add one space before {

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: vijaya/html/assignment5/question1.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  h2{
+   color:white;
+    text-align:right;
+  }
+  a:link {
+text-decoration:blueviolet;
+}
+ul{
+ list-style-type: square;
+ }
```
Comment (Pawan1005) [inline vijaya/html/assignment5/question1.css]: add space before {

**Review by Pawan1005 — APPROVED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add space before {

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**